### ISSUE TITLE
Not all Longhorn UI buttons are functioning (Example: backups)

### ISSUE THREAD

**Issue by gaktive** [rancher/dashboard#5024]
Internal reference: SURE-3990
Reported in 2.6.2 with Longhorn 1.2.2

Issue description:
Backups by selecting the volume using a checkbox or by entering the volume and then using the backup button do not always work.

Business impact:
The UI is not usable for safely making/controlling and working with backups.

Troubleshooting steps:

- Backups by selecting the volume using a checkbox or by entering the volume and then using the backup button do not always work.
- We do not see any POST calls happening in the first few attempts.
- After 3rd or 4th continuous triggers, the POST calls get generated.
- The first GET call logs in the longhorn-system namespace for longhorn-ui pod logs the following:
```
[06/Jan/2022:09:40:14 +0000] "GET /v1/volumes HTTP/1.1" 200 71197 "https://$SERVER_NAME/k8s/clusters/local/api/v1/namespaces/longhorn-system/services/http:longhorn-frontend:80/proxy/ <https://$SERVER_NAME/k8s/clusters/local/api/v1/namespaces/longhorn-system/services/http:longhorn-frontend:80/proxy/> " "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:91.0) Gecko/20100101 Firefox/91.0"
10.42.0.7 - - [06/Jan/2022:09:40:14 +0000] "HEAD /healthz HTTP/1.1" 200 0 "-" "-"
10.42.0.7 - - [06/Jan/2022:09:40:16 +0000] "HEAD /healthz HTTP/1.1" 200 0 "-" "-"
10.42.0.7 - - [06/Jan/2022:09:40:17 +0000] "HEAD /healthz HTTP/1.1" 200 0 "-" "-"
```

- The second, third, or fourth call which completes successfully, makes triggers the POST }}call after the {{GET calls.
```
10.42.0.1 - - [06/Jan/2022:09:42:34 +0000] "GET /v1/volumes HTTP/1.1" 200 71190 "https://$SERVER_NAME/k8s/clusters/local/api/v1/namespaces/longhorn-system/services/http:longhorn-frontend:80/proxy/ <https://SERVER_NAME/k8s/clusters/local/api/v1/namespaces/longhorn-system/services/http:longhorn-frontend:80/proxy/> " "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:91.0) Gecko/20100101 Firefox/91.0"
10.42.0.7 - - [06/Jan/2022:09:42:34 +0000] "HEAD /healthz HTTP/1.1" 200 0 "-" "-"
10.42.0.7 - - [06/Jan/2022:09:42:36 +0000] "HEAD /healthz HTTP/1.1" 200 0 "-" "-"
10.42.0.7 - - [06/Jan/2022:09:42:37 +0000] "HEAD /healthz HTTP/1.1" 200 0 "-" "-"
10.42.0.1 - - [06/Jan/2022:09:42:38 +0000] "POST /v1/volumes/pvc-e390eb92-8d9c-4ce3-8f9a-38b7217c94a0?action=snapshotCreate HTTP/1.1" 200 392 "https://$SERVER_NAME/k8s/clusters/local/api/v1/namespaces/longhorn-system/services/http:longhorn-frontend:80/proxy/ <https://$SERVER_NAME/k8s/clusters/local/api/v1/namespaces/longhorn-system/services/http:longhorn-frontend:80/proxy/> " "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:91.0) Gecko/20100101 Firefox/91.0"
10.42.0.1 - - [06/Jan/2022:09:42:38 +0000] "POST /v1/volumes/pvc-e390eb92-8d9c-4ce3-8f9a-38b7217c94a0?action=snapshotBackup HTTP/1.1" 200 7324 "https://$SERVER_NAME/k8s/clusters/local/api/v1/namespaces/longhorn-system/services/http:longhorn-frontend:80/proxy/ <https://$SERVER_NAME/k8s/clusters/local/api/v1/namespaces/longhorn-system/services/http:longhorn-frontend:80/proxy/> " "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:91.0) Gecko/20100101 Firefox/91.0"
10.42.0.7 - - [06/Jan/2022:09:42:39 +0000] "HEAD /healthz HTTP/1.1" 200 0 "-" "-"
10.42.0.7 - - [06/Jan/2022:09:42:41 +0000] "HEAD /healthz HTTP/1.1" 200 0 "-" "-"
10.42.0.7 - - [06/Jan/2022:09:42:42 +0000] "HEAD /healthz HTTP/1.1" 200 0 "-" "-"
```

- Once the volumes are selected and when the backups for them are triggered, they do not succeed either.
- For an unsuccessful backup, an API call does not perform the backup of volumes but rather makes a GET API call.
- However, for a successful API call, we see the POST request to perform backups.
- When the backups are not successful, we do not see any errors related to CORS in the console logs.
- We applied the longhorn MIDDLEWARE and INGRESS objects since they are using Traefik Ingress controllers but that does not seem to fix the issue.

Repro steps:

Local repro attempted but things looked fine.

Workaround:
The workaround is to keep trying until the backups work. However, scheduled backups do not work and the user does not have any workaround for scheduled backups that are failing. 

Actual behavior:
Backups using Longhorn UI are not consistently functional. 

Expected behavior:
Backups using Longhorn UI should be functional using available backup options.

**Comment by gaktive** [comment]
Workaround other than "keep trying" is being requested since the scheduled ones are more important. Checking on additional environment information.

**Comment by gaktive** [comment]
This is being handled by the Longhorn team in https://github.com/longhorn/longhorn/issues/3191. Closing this ticket as a duplicate; they have a fix in their v1.2.3 but if workarounds are needed, follow up with the Longhorn folks.
